vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -304,6 +310,62 @@ void FrameworkElementViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate,
         }
         AnnounceLiveRegionChangedIfNeeded(element);
       }
+      else if (propertyName == "accessibilityRole")
+      {
+        if (propertyValue.isString())
+        {
+          const std::string& role = propertyValue.getString();
+          if (role == "none")
+            DynamicAutomationProperties::SetAccessibilityRole(element, winrt::react::uwp::AccessibilityRoles::None);
```

Comment:
So only <View>  viewpanel/viewcontrol create the dynamicautomationpeer, if you set these properties on some other element type (an image or something) - whta happens?  the property is set but nothing is looking for it until we update everything to create custom automation peers?

Reply:
Yes. We can update other controls to use DynamicAutomationPeer (unfortunately via subclassing, but that's the only method XAML gives us) but now third party developers creating custom view managers can take advantage of our accessibility work.